packages/offix-scheduler/src/store/OfflineStore.ts

```diff
@@ -35,50 +36,57 @@ export class OfflineStore<T> {
    * @param entry - the entry to be saved
    */
   public async saveEntry(entry: QueueEntryOperation<T>) {
-    const serialized = this.serializer.serializeForStorage(entry);
-    const offlineKey = this.getOfflineKey(entry.qid);
-    // only add the offline key to the arrray if it's not already there
-    if (!this.arrayOfKeys.includes(offlineKey)) {
-      this.arrayOfKeys.push(offlineKey);
-      await this.storage.setItem(this.offlineMetaKey, this.arrayOfKeys);
-    }
-    await this.storage.setItem(offlineKey, serialized);
+    
+      const serialized = this.serializer.serializeForStorage(entry);
+      const offlineKey = this.getOfflineKey(entry.qid);
+      // only add the offline key to the arrray if it's not already there
+      if (!this.arrayOfKeys.includes(offlineKey)) {
+        this.arrayOfKeys.push(offlineKey);
+        await this.storage.setItem(this.offlineMetaKey, this.arrayOfKeys);
+      
+      await this.storage.setItem(offlineKey, serialized);
+      debug("Saved Item");
```

Comment:
Trick is to actually serialize item :D

Reply:
You can add items in as added arguments. So,
`debug("Saved item", serialized)` - note if we change the name of the logger function, use that
name instead of `debug`.